packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -69,15 +71,40 @@ type BaseButtonCommonProps = {
   accessibilityProps?: Partial<AccessibilityProps>;
   variant?: 'primary' | 'secondary' | 'tertiary';
   color?: 'primary' | 'white' | 'positive' | 'negative' | 'notice' | 'information' | 'neutral';
+  isPressAnimationDisabled?: boolean;
 } & TestID &
   StyledPropsBlade &
   BladeCommonEvents;
 
+type ButtonImgProps = {
+  /**
+   * Custom image source
+   */
+  src: string;
+  /**
+   * The `alt` attribute for the `img` element
+   */
+  alt?: string;
+  /**
+   * The `srcSet` attribute for the `img` element, useful for responsive images.
+   */
+  srcSet?: string;
```

Comment:
Is it expected for Avatar to be built on top of BaseButton? aren't they 2 different components 🤔